spec/swagger.yaml

```diff
@@ -124,26 +124,20 @@ tags:
+      The Job Batches APIs provide a wrapper around Jobs and Files that simplifies workin with them. Users can more easily combine uploading files and adding them to Jobs.
+      Generally, adding multiple files to a Job forces the user to implement polling. This is because The Jobs API expects a file to be uploaded and fully processed before being added to a Job. In other words the Add File to Job API is *not* actually an API ~upload~, it just associates a file that was uploaded separately, to the Job. That process of uploading a file to Smartling is asynchronous.  The file needs to be fully processed before it's ready to be worked with, for example, before it's added to a Job. Lastly a Job can't be authorized until any background tasks for its component parts are completed. Job Batches simplifies these complexities.
+      The Job Batches API utilizes internal Smartling events to optimize the process of adding uploading files and adding them to Jobs.
+      The flow for the API user is as follows:
+      1. Client identifies the Job to which they want to add files. They can create a new Job or get the ID for existing one. As long as it is *not* currently being processed with another Batch)
+      2. Client creates a new Batch for the Job to which they want to add files.
+      3. Client  uploads files using the Job Batch Upload endpoint, optionally specifying the target locales list for each file. This endpoint has the same parameters as a generic file upload to a Smartling project for specifying exactly how the file should be handled by Smartling.
+      4. After "uploading" all the files, the client finalizes the Batch by calling the "execute" command for it. Note that the client doesn't need to check the status for the individual files.
+      5. The Batch execution process handles waiting for all file upload processing to be completed, actually "adding" them to the Job (associating the File with the Job) and if it was specified, authorizes the Job.
+      6. Clients can check the Batch status to understand the status of the individual files that were uploaded.  Batch status will indicate if they were uploaded successfully and provide statistics about the strings for the files added to the Job, or if not, what errors happened while uploading the file.
+      After a Batch is completed it can't be reused.
+      **Limitations for Job Batches**
+      - Correct handling of multiple Batches in parallel is not guaranteed. Creating and working with a new Batch should be done only after previous Batch reached "completed" state. Use the List Batches call on the Job in question to confirm no Batches are currently open (any state other than "Completed").
+      - The entire Smartling system is asynchronous and there is currently no way to "lock" a Job. So this service cannot guarantee correct processing of a Batch it's corresponding Job or the Files that are part of the Batch are modified using the Smartling Dashboard UI or with another integration.
```

Comment:
Unclear:
So this service cannot guarantee correct processing of a Batch it's corresponding Job or the Files that are part of the Batch are modified using the Smartling Dashboard UI or with another integration.

Reply:
How about: "As such, the Jobs Batch service cannot guarantee the result of processing a Batch will be exactly as expected.  If its corresponding Job or the Files that are part of the Batch have been modified in the Smartling Dashboard UI or with another integration, the results of executing a Batch could be affected."

Reply:
Looks good.